LeadKit/LeadKit/Controllers/TableViewController.swift

```diff
@@ -0,0 +1,143 @@
+//
+//  TableViewController.swift
+//  LeadKit
+//
+//  Created by Ivan Smolin on 30/05/16.
+//  Copyright © 2016 Touch Instinct. All rights reserved.
+//
+
+import UIKit
+
+public class TableViewController: UITableViewController, CellsControllerProtocol {
+
+    private var heightCache: [NSIndexPath: CGFloat] = [:]
```

Comment:
Do you store heights of cells in ViewController? It's not good architecture, maybe it's better to store heights in cells view models?